docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -9,6 +9,14 @@
     "This tutorial will cover the information you can access on your account and related jobs and targets using `cirq-superstaq`."
```

Comment:
<p>fwiw i don't think the "ready" or "deleted" statuses can actually occur</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Ah I see, I just based it off:
https://github.com/Infleqtion/client-superstaq/blob/b512cf185b62591544e3332b0eef79651f9892c2/cirq-superstaq/cirq_superstaq/job.py#L107-L115
for completeness

Reply:
yeah, we also include them in e.g. `css.Job.ALL_STATES`.. so definitely outside the scope of this pr to correct. your call on whether we include them in the notebook until then

otherwise lgtm!